src/components/AddressInfo.css

```
@@ -34,10 +32,25 @@
     background-repeat: no-repeat;
     background-position: center;
     background-size: contain;
-    background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" fill="none" width="146" height="146"><path d="M113.829 16.20974L142.47 65.7903c2.578 4.461 2.578 9.9584 0 14.4194L113.829 129.79c-2.57 4.461-7.335 7.21-12.488 7.21H44.0555c-5.1529 0-9.918-2.749-12.4877-7.21L2.92725 80.2097c-2.56966698-4.461-2.56966698-9.9584 0-14.4194L31.5723 16.20974C34.151 11.74872 38.9116 9 44.06 9h57.281c5.153 0 9.918 2.74872 12.488 7.20974z" clip-rule="evenodd" stroke="%231f2348" stroke-width="2" opacity=".2"/></svg>');
     z-index: 1;
 }
 
+.address-info .identicon.clip > img {
+    clip-path: url(#nimiq-rounded-hexagon);
+}
+
+.address-info .identicon.clip::after {
+    background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" fill="none" width="146" height="146"><path d="M113.829 16.20974L142.47 65.7903c2.578 4.461 2.578 9.9584 0 14.4194L113.829 129.79c-2.57 4.461-7.335 7.21-12.488 7.21H44.0555c-5.1529 0-9.918-2.749-12.4877-7.21L2.92725 80.2097c-2.56966698-4.461-2.56966698-9.9584 0-14.4194L31.5723 16.20974C34.151 11.74872 38.9116 9 44.06 9h57.281c5.153 0 9.918 2.74872 12.488 7.20974z" clip-rule="evenodd" stroke="%231f2348" stroke-width="2" opacity=".2"/></svg>');
+}
+
+.address-info .identicon.clip-circle > img {
+    clip-path: url(#nimiq-circle);
```

I don't think you need a clip path for a circle...
```suggestion
    border-radius: 50%;
```

---

Also, if using `clip-path` `circle` can be used directly as basic shape: https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path